Examples/Examples-iOS/IGListKitMessageExample/MessagesViewController.swift

```diff
@@ -16,6 +16,7 @@ import UIKit
 import Messages
 import IGListKit
 
+@available(iOSApplicationExtension 10.0, *)
```

Comment:
Do we know why this is suddenly required? Assuming the target version hasn't been changed?

Reply:
No idea tbh, failed to build so just clicked "fix" haha